crowdfunding_compassion/controllers/project_controller.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime
+
+from babel.dates import format_timedelta
+
+from odoo.http import request, route, Controller
+
+
+class ProjectController(Controller):
+    # Demo route used to display demo project
+    @route("/project/", auth="public", website=True)
```

Comment:
Maybe use a less generic route than `/project` for a demo page. `/project/demo` would be better. But this will make the page available to everyone so not sure it's good to keep it like that. Maybe add a TODO comment to remove this route after we finish with the developement?

Reply:
this route is not used anymore, I will just remove it